examples/opensearchserverless/main.tf

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+# Require Terraform 0.12 or greater
+terraform {
+  required_version = ">= 0.12"
+}
+
+# Set AWS provider region
+provider "aws" {
+  region = var.aws_region
+}
+
+# Creates an encryption security policy
+resource "aws_opensearchserverless_security_policy" "encryption_policy" {
+  name        = "example-encryption-policy"
+  type        = "encryption"
+  description = "encryption policy for ${var.collection_name}"
+  policy = jsonencode({
+    Rules = [
+      {
+        Resource = [
+          "collection/${var.collection_name}"
+        ],
+        ResourceType = "collection"
+      }
+    ],
+    AWSOwnedKey = true
+  })
+}
+
+# Creates a collection
+resource "aws_opensearchserverless_collection" "collection" {
+  name = var.collection_name
+
+  depends_on = [aws_opensearchserverless_security_policy.encryption_policy]
+}
+
+# Creates a network security policy
```

Comment:
We don't say network security policy, rather we say network policy in our docs, Can you change it to that and remove "security"